Assets/Scripts/BlockBehaviour.cs

```diff
@@ -5,12 +5,26 @@
 
 public class BlockBehaviour : MonoBehaviour
 {
+    /*
+     * Faces
+     */
+    public BlockFaceBehaviour Top;
+    public BlockFaceBehaviour Bottom;
+    public BlockFaceBehaviour Front;
+    public BlockFaceBehaviour Back;
+    public BlockFaceBehaviour Left;
+    public BlockFaceBehaviour Right;
+
+    [Range(0f, 1f)]
+    public float SkinToLengthRatio = 0.1f;

```

Comment:
Why is it called skin? Is there more descriptive name?

Reply:
Raycast are fired from within the blocks. Thus we are firing from under the block's "skin".